### PR TITLE
enterprise: first stab at kubernetes documentation

### PR DESCRIPTION
**Pull request by iaguis**

### CODE REVIEW

File: enterprise/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+## Installing Private Packagist Enterprise from scratch
+
+### Requirements
+
+* A Linux Server
+  * OS
+    * Ubuntu 16.04.3 - 16.04.5
+    * Red Hat Enterprise Linux 7.4 - 7.5
+    * CentOS 7.4 - 7.5
```
Comment (naderman) [inline enterprise/kubernetes.md]: Where did this list come from?

Comment (iaguis) [reply enterprise/kubernetes.md]: Copied it from here: https://help.replicated.com/docs/kubernetes/customer-installations/supported-operating-systems/

Comment (naderman) [reply enterprise/kubernetes.md]: Ah thanks, that appears to be for the "easy install script" only. Other distributions and versions etc. are supported for manual installation, too. So I'd rather not include the list here specifically.

File: enterprise/kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  * At least 40GB disk space
+  * Ports 443 and 8800 must be accessible
+  * Must be reachable at the chosen domain names from your local machine
+
+### Installation
+
+The easiest way to install Replicated Kubernetes is the easy install script. You can run it using the following two commands:
+
+```
+curl -sSL -o install.sh https://get.replicated.com/kubernetes-init
```
Comment (naderman) [inline enterprise/kubernetes.md]: Think the URL here should be https://get.replicated.com/kubernetes-init/privatepackagist/stable similar to what we have for the Native Scheduler, this isn't really well documented by Replicated, but it ensures the version of Replicated gets installed which we are using on our latest stable release.

Comment (iaguis) [reply enterprise/kubernetes.md]: If I try https://get.replicated.com/kubernetes-init/privatepackagist/stable I get this at the bottom:

```bash
printf "${RED}
            404: Not Found
            There was an error processing the request to generate the installation script

                Request path: $PATH

            Please visit https://help.replicated.com/ for installation guides and documentation${NC}\n\n\
"

exit 1
```

so I don't think that works?

Comment (naderman) [reply enterprise/kubernetes.md]: Yeah might be for the native scheduler only, so let's just leave it like this, I may check with them at some later point.

Comment (naderman) [reply enterprise/kubernetes.md]: Actually found the correct instructions on their admin UI:

```
curl -sSL -o install.sh https://get.replicated.com/privatepackagistkubernetes/stable/kubernetes-init
sudo bash install.sh
```

URL just looked a bit different than expected ;-)

**Comment by glaubinix** [comment]
Continues https://github.com/packagist/private-packagist-docs/pull/67

**Pull request closed without merging**